Proton/Sources/Core/PRTextStorage.m

```diff
@@ -69,48 +63,44 @@ - (id)attribute:(NSAttributedStringKey)attrName atIndex:(NSUInteger)location eff
     return [self.storage attribute:attrName atIndex:location effectiveRange:range];
 }
 
-- (NSDictionary<NSString *,id> *)attributesAtIndex:(NSUInteger)location effectiveRange:(NSRangePointer)effectiveRange {
-//    if (location <= self.storage.length) {
```

Comment:
Happy to bring back

Reply:
From memory, this stopped a crash in our app. But maybe not having any bridging will avoid it? Not sure. We'll have to keep an eye on this.

Reason it is commented out though is because this would hang because of a deadlock somewhere.